server-src/src/main/java/com/itt/kmt/model/ArticleApproval.java

```diff
@@ -0,0 +1,56 @@
+package com.itt.kmt.model;
+
+import lombok.Data;
+
+/**
+ * This class represents a ArticleApproval.
+ * 
+ * @author Ashish Y
+ */
+@Data
+public class ArticleApproval {
+
+    /**
+     * approval Status of the Article.
+     */
+    private Boolean approvalStatus =  false;

```

Comment:
Either make approvalStatus should be a enum or changed the field name as "approved"